services/wallet/activity/service.go

```diff
@@ -317,7 +317,7 @@ func (s *Service) GetOldestTimestampAsync(requestID int32, addresses []common.Ad
 		if errors.Is(err, context.Canceled) || errors.Is(err, async.ErrTaskOverwritten) {
 			res.ErrorCode = ErrorCodeTaskCanceled
 		} else if err == nil {
-			res.Timestamp = result.(int64)
+			res.Timestamp = int64(result.(uint64))
```

Comment:
@Cuteivist are we sure that the value is always `uint64`, or it can be `int64` as well?

Maybe better to handle it something like:
```
if value, ok := result.(uint64); ok {
    res.Timestamp = int64(value)
} else if value, ok := result.(int64); !ok {
    return
}
```